DataProcessing/datax-core/src/main/scala/datax/constants/BlobProperties.scala

```diff
@@ -7,4 +7,6 @@ package datax.constants
 object BlobProperties {
   // Define constants for blobs
   val BlobHostPath = ".blob.core.windows.net"
+  val AzureStorageJarPath = "/datax/bin/azure-storage-3.1.0.jar"
```

Comment:
Lets move the jar name to external config and avoid hardcoded names in the source. This will make updating the version easy.

Reply:
Moved to env variable